recipes/pggb/build.sh

```diff
@@ -7,3 +7,5 @@ sed -i ${n}'a\if [ $show_version == true ]; then echo "pggb '$PKG_VERSION'"; exi
 
 mkdir -p $PREFIX/bin
 mv pggb $PREFIX/bin
+mv partition-before-pggb $PREFIX/bin
+mv scripts/*.py $PREFIX/bin
```

Comment:
:fire:

Reply:
@AndreaGuarracino This should have brought the files in. I am confused.

Reply:
9, it is moving the files, but `partition-before-pggb` wants the `scripts` folder.